### PR TITLE
HAWQ-365. Improve DEBUG level log for counting resource queue active connections

### PR DESCRIPTION
**Pull request by jiny2**

### CODE REVIEW

**Comment by yaoj2** [comment]
＋1

**Comment by radarwave** [comment]
LGTM. +1

File: src/backend/resourcemanager/resqueuemanager.c [hunk 1]
```diff
@@ -2142,8 +2142,10 @@ int registerConnectionByUserID(ConnectionTrack  conntrack,
 
 	transformConnectionTrackProgress(conntrack, CONN_PP_REGISTER_DONE);
 
-	elog(LOG, "ConnID %d. Connection is registered.", conntrack->ConnID);
-
+	elog(DEBUG3, "Resource queue %s has %d connections after registering "
+				 "connection.",
+				 queuetrack->QueueInfo->Name,
+				 queuetrack->CurConnCounter);
```
Comment (huor) [inline src/backend/resourcemanager/resqueuemanager.c]: It it better to add current ConnID here:
elog(DEBUG3, "Resource queue %s has %d connections after registering "
                                               "connection ConnID %d.",
                                               queuetrack->QueueInfo->Name,
                                               queuetrack->CurConnCounter,
                                               conntrack->ConnID);

File: src/backend/resourcemanager/resqueuemanager.c [hunk 2]
```diff
@@ -2216,6 +2223,9 @@ int acquireResourceFromResQueMgr(ConnectionTrack  conntrack,
 	int						res			= FUNC_RETURN_OK;
 	DynResourceQueueTrack	queuetrack	= conntrack->QueueTrack;
 
+	elog(LOG, "ConnID %d. Expect query resource for session "INT64_FORMAT,
```
Comment (huor) [inline src/backend/resourcemanager/resqueuemanager.c]: Two values needs to be provided here for "%d" and "INT64_FORMAT". But here is only "conntrack->SessionID"

**Comment by huor** [comment]
The overall fix looks good to me. +1

**Pull request closed without merging**